include/readdy/model/Particle.h

```diff
@@ -57,16 +57,16 @@ namespace readdy::model {
 class Particle {
 public:
 
-    using id_type = unsigned long;
-    using pos_type = Vec3;
-    using type_type = ParticleTypeId;
+    using Id = unsigned long;
```

Comment:
there is a `Particle::id_type` which could be moved to the common type definitions as `ParticleId` or something